### ISSUE TITLE
Shorter table headers

### ISSUE THREAD

**Issue by msalak-scia** [StructuralAnalysisFormat/gitbookdocumentation#58]
Table headers are longer than necessary, sometimes defining the width of the whole column. 
![image](https://user-images.githubusercontent.com/89633938/145399660-9c3a157e-a762-45a7-8bc2-2e9d5266a3c2.png)

Change the names to:

Column header | Data type | Example / enum definition | Required value | Description
(Alt. Row Header)

**Comment by msalak-scia** [comment]
internal reference number 5371

**Comment by dschuermans** [comment]
As mentioned earlier, instead of putting the whole enum definition into the table, it might also be an option to have a separate page under annexes that lists the different enumerations available.

This will:
* Reduce the height of the row, making more of the table fit on the page
* Have the enum defined only once in the documentation
  * Less chance for errors, due to typo's or mismatches
  * Only 1 place to look for and update the enum definition if it ever changes.
  
The only "downside" to this would be that you wont have the enum definition in the table, but that might be solved by something like this I found on RTD docs:
![image](https://user-images.githubusercontent.com/6634696/145539496-f5b877fe-b5b1-4171-bcf7-5bd747de5005.png)

If you hover over the link, it'll open a pop-up which presumably lists the content of another page in the docs.
Something worth investigating? @msalak-scia @sonkaste17 


Secondly,

* "Column header" or "Row header" can just be replaced with "Header"?
* "Type of data" might just as well be "Type"?
* I'd propose the following order, so that the more important columns are up front:
  | Header | Type | Required | Description | Example |

**Comment by dschuermans** [comment]
For that popup thing to work, we'd have to try something similar to this:

> check out :ref:`connected-accounts:permissions for connected accounts`.

They have, in their doc root folder a file called "connected-accounts.rst" with a title in there "permissions for connected accounts"

Although this may only work with rst files and not such much markdown files?

**Comment by dschuermans** [comment]
https://sphinx-hoverxref.readthedocs.io/en/latest/

**Comment by msalak-scia** [comment]
We discussed that it would be nice to have all the values of enums directly inside the table. Sometimes it is necessary for the orientation in tables. In case required value of something is 'yes if other enum = ...' then it is nice to look at the enum a see all the possible option, to get the idea how it works. What do you think? @dschuermans

**Comment by dschuermans** [comment]
@msalak-scia Then why do it for most enums and not for all? 😅 
You're free to do whatever you want with this, I just offered up a solution to keep the tables which define the columns that should/may be present on the sheet as concise as possible.
